src/style.css

```diff
@@ -1,3 +1,72 @@
-body {
-  margin: 0;
-}
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.nav {
+  box-shadow: 0 2px 4px 0 rgba(0,0,0, 0.3);
+}
+
+.nav__wrapper {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__logo {
+  align-self: center;
+  justify-content: center;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;

```

Comment:
Move all `rest type` styles to the top of the file and apply them to the tag name. Later all such styles will lay in reset.css file